lib/uklock/mutex.c

```diff
@@ -1,8 +1,39 @@
 #include <uk/mutex.h>
 
+#ifdef CONFIG_LIBUKLOCK_MUTEX_METRICS
+#include <uk/init.h>
+
+struct uk_mutex_metrics _uk_mutex_metrics;
```

Comment:
```suggestion
struct uk_mutex_metrics _uk_mutex_metrics = { 0 };
```

Easily initialize everything to 0 so there is no need for `memset`